IPython/terminal/debugger.py

```diff
@@ -87,7 +87,7 @@ def set_trace(frame=None):
 if __name__ == '__main__':
     import pdb
     # IPython.core.debugger.Pdb.trace_dispatch shall not catch
-    # bdb.BdbQuit. When started through __main__ and an exeption
+    # bdb.BdbQuit. When started through __main__ and an exception
     # happend after hitting "c", this is needed in order to
```

Comment:
Thanks for fixing the typo :cookie: I will let @takluyver or @fperez merge. Thanks for the contribution.